uniflow/pipeline.py

```diff
@@ -4,17 +4,29 @@
 from threading import Thread
 from typing import Any, List, Mapping
 
-from uniflow.config import PipelineConfig
-from uniflow.extract.client import Client as ExtractClient
-from uniflow.schema import Context
-from uniflow.transform.client import Client as TransformClient
+from uniflow.flow.config import PipelineConfig
+from uniflow.flow.client import ExtractClient, TransformClient
+from uniflow.op.prompt_schema import Context
 
 
-class Pipeline:
-    """Queue-Based Pipeline for Uniflow"""
+class MultiProcessPipeline:
```

Comment:
MultiProcessPipeline is a very confusing name. `MultiStepPipeline` or `MultiStagePipeline` will be a better name.

Reply:
Fixed